.github/workflows/integration_tests.yaml

```diff
@@ -0,0 +1,65 @@
+name: Run integration tests
+on:
+  workflow_call:
+    secrets:
+      CICD_OIDC_ROLE:
+        required: true
+      GH_PSGA_SYSTEM_PAT:
+        required: true
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  start-runner:
+    uses: Congenica/psga-reusable-workflows/.github/workflows/ec2-runner-start.yaml@main
+    secrets:
+      role-to-assume: ${{ secrets.CICD_OIDC_ROLE }}
+      github-token: ${{ secrets.GH_PSGA_SYSTEM_PAT }}
+    with:
+      aws-region: ${{ vars.AWS_REGION }}
+      ec2-instance-type: m5.2xlarge
+      subnet-id: ${{ vars.GH_RUNNER_SUBNET_ID }}
+      security-group-id: ${{ vars.GH_RUNNER_SECURITY_GROUP_ID }}
+      iam-role-name: ${{ vars.GH_RUNNER_IAM_ROLE_NAME }}
+      env_type: ${{ inputs.actions_env }}
+
+  integration_tests:
+    name: Integration Tests
+    needs: start-runner # required to start the main job when the runner is ready
+    runs-on: ${{ needs.start-runner.outputs.label }}
+    environment: ${{ inputs.actions_env }}
+    steps:
+      #   - name: Check out repo
+      #     uses: actions/checkout@v4
+
+      #   - name: Configure AWS Credentials
+      #     id: creds
+      #     uses: aws-actions/configure-aws-credentials@v4
+      #     with:
+      #       aws-region: eu-west-2
+      #       role-to-assume: ${{ secrets.CICD_OIDC_ROLE }}
+      #       role-session-name: cicd-role-session
+      #       mask-aws-account-id: false
+
+      #   - name: Login to dev Amazon ECR
+      #     uses: aws-actions/amazon-ecr-login@v2
+
+      - name: Run tests
+        run: |
+          echo "run 3x integration tests here, one for each image"
```

Comment:
As discussed offline, this is fine for this PR but we need to make sure this turns into actual integration tests to avoid giving false confidence that integration tests are passing.